packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -383,6 +395,12 @@ class _DragTargetState<T> extends State<DragTarget<T>> {
   final List<_DragAvatar<T>> _candidateAvatars = <_DragAvatar<T>>[];
   final List<_DragAvatar<dynamic>> _rejectedAvatars = <_DragAvatar<dynamic>>[];
 
+  void onDrag(_DragAvatar<dynamic> avatar, Offset lastOffset) {
```

Comment:
Had to change it back to dynamic. With T, I get this exception when running the test 'Drag and drop - allow pass thru of unaccepted data twice test':
type '_DragAvatar<DragTargetData>' is not a subtype of type '_DragAvatar<ExtendedDragTargetData>' of
'avatar'

This test and the test 'Drag and drop - allow pass thru of unaccepted data test' require that the type is dynamic.